jekyll/_cci2/docker-layer-caching.md

```diff
@@ -1,58 +1,85 @@
+Use Docker layer caching (DLC) to reduce Docker image build times on CircleCI. DLC is available on all CircleCI plans, and credits are charged per job run.
 
-## Overview
-{: #overview }
+## Introduction
+{: #introduction }
 
-Docker layer caching (DLC) is beneficial if building Docker images is a regular part of your CI/CD process. DLC will save image layers created within your jobs, rather than impact the actual container used to run your job.
+Docker layer caching (DLC) is beneficial if building Docker images is a regular part of your CI/CD process. DLC saves image layers created within your jobs, using a [_sparse file_](https://en.wikipedia.org/wiki/Sparse_file).
 
-DLC caches the individual layers of any Docker images built during your CircleCI jobs, and then reuses unchanged image layers on subsequent CircleCI runs, rather than rebuilding the entire image every time. In short, the less your Dockerfiles change from commit to commit, the faster your image-building steps will run.
+DLC caches the individual layers of any Docker images built during your CircleCI jobs, and then reuses unchanged image layers on subsequent job runs, rather than rebuilding the entire image every time. In short, the less your Dockerfiles change from commit to commit, the less time your image-building jobs will take to run.
 
 Docker layer caching can be used with both the `machine` executor and in the [remote Docker environment](/docs/building-docker-images/) (`setup_remote_docker`).
 
-The underlying implementation of DLC is in the process of being updated. **There is no action required from users.** All further content on this page refers to the implementation of DLC that is in the process of being phased out. Once all jobs have been migrated to the new implementation, the content currently on this page will become outdated and will be replaced with information based on the new architecture.
-<br>
-<br>
-Visit the [Discuss post](https://discuss.circleci.com/t/fyi-small-dlc-update-no-action-required/44614) to learn more details regarding the new architecture and to follow updates regarding the rollout.
-{: class="alert alert-info"}
+## Quickstart
+{: #quickstart }
 
-### Limitations
-{: #limitations }
+### Remote Docker environment
+{: #remote-docker-environment }
+
+To use DLC with the Docker execution environment, you will need to configure your job to run in the Remote Docker Environment. To do this, add `docker_layer_caching: true` under the `setup_remote_docker` key in your [`.circleci/config.yml`](/docs/configuration-reference/) file:
+
+```yaml
+version: 2.1
+
+jobs:
+ build:
+    docker:
+      - image: cimg/base:2023.04
+
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: true
+```
+
+Every layer built in a previous job will be accessible in the Remote Docker Environment. However, in some cases your job may run in a clean environment, even if the configuration specifies `docker_layer_caching: true`.
```

Comment:
This to me makes it feel like we are making a promise to customer that we will keep all layers for the customer which is not true. We do pruning on layers to reduce the size of cache and try our best to store as much useful pieces as possible. We did this in old world too. cc @sebastian-lerner

Reply:
can we get rid of this sentence too @rosieyohannan ?